lib/src/directory/localstorage_io.dart

```diff
@@ -79,7 +79,7 @@ class DirUtils implements LocalStorageImpl {
     _data = data;
     storage.add(data);
     File _file = await _getFile();
-    _file.writeAsString(json.encode(data), flush: true);
+    _file.writeAsString(((data != null && data.isNotEmpty) ? json.encode(data) : data), flush: true);
```

Comment:
could you please provide an example in which `data` could be `null` or `""` here?

Reply:
nevermind, I was able to reproduce the issue